static/js/lib/redux_props_test.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import { assert } from "chai"
+import sinon from "sinon"
+
+import { mergeAndInjectProps } from "./redux_props"
+
+describe("redux_props", () => {
+  let sandbox
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create()
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe("mergeAndInjectProps", () => {
+    it("should pass stateProps, dispatchProps, and ownProps into the passed method", () => {
```

Comment:
Can you add some overlapping props verifying the order the spread operator gets applied?